cms/management/commands/subcommands/base.py

```diff
@@ -36,7 +36,7 @@ def add_builtin_arguments(parser):
         help="Don't colorize the command output.")
     parser.add_argument('--force-color', action='store_true', dest='force_color', default=False,
         help="Colorize the command output.")
-    if DJANGO_3_0 or DJANGO_3_1 or DJANGO_3_2:
+    if DJANGO_VERSION > '3.0':
```

Comment:
Since `DJANGO_2_2` is the last version without the skip checks command supported can this be `if not DJANGO_2_2:`? I agree that it is a long time before `DJANGO_VERSION > '3.0'` might fail incorrectly but I guess it also makes it transparent that the if can be removed all together once Django 2.2 is history.

Reply:
Good point, fixed.